include/swift/Option/Options.td

```diff
@@ -428,6 +428,9 @@ def disable_bridging_pch : Flag<["-"], "disable-bridging-pch">,
 def enable_experimental_differentiable_programming : Flag<["-"], "enable-experimental-differentiable-programming">,
   Flags<[FrontendOption]>,
   HelpText<"Enable experimental differentiable programming features">;
+def enable_experimental_forward_mode_differentiation : Flag<["-"], "enable-experimental-forward-mode-differentiation">,
```

Comment:
Please surround newly added code in `// SWIFT_ENABLE_TENSORFLOW` and `// SWIFT_ENABLE_TENSORFLOW END` comments - I believe TableGen supports `//` and `/* ... */` comments.

Please also add: `// NOTE: This flag will be removed when JVP/differential generation is robust.`

```cpp
// SWIFT_ENABLE_TENSORFLOW
// NOTE: This flag will be removed when JVP/differential generation is robust.
def enable_experimental_forward_mode_differentiation : Flag<["-"], "enable-experimental-forward-mode-differentiation">,
  Flags<[FrontendOption]>,
  HelpText<"Enable experimental forward mode differentiation">;
// SWIFT_ENABLE_TENSORFLOW END
```